examples/RazorWeb/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
What does this file has to do with the PR? Did you maybe commit this by mistake?

Reply:
Nothing, it shouldn't be in the commit.